### PR TITLE
ResNet Docs + CIFAR10 Example

### PR DESCRIPTION
**Pull request by alexander-g**
- Added docstrings for ResNet (#117)
- Added low resolution version of ResNet as described in subsection 4.2 of the original paper for small images like in CIFAR10
- Added CIFAR10 example which uses a low resolution ResNet18 to achieve 95% accuracy

TODO:
- mkdocs

### CODE REVIEW

**Comment by alexander-g** [comment]
- mkdocs

Requesting review @cgarciae

File: elegy/nets/resnet.py [hunk 1]
```diff
@@ -2,49 +2,80 @@
 
 import jax, jax.numpy as jnp
 from elegy import module, nn
+import typing as tp
+
+
+__all__ = ["ResNet18", "ResNet34", "ResNet50", "ResNet101", "ResNet152", "ResNet200"]
```
Comment (cgarciae) [inline elegy/nets/resnet.py:8]: Should we add the base `ResNet` as well?

Comment (alexander-g) [reply elegy/nets/resnet.py]: Done
